doc/user_guide/interactions.rst

```diff
@@ -476,8 +636,8 @@ points based on whether they are smaller or larger than the value:
    })
 
    slider = alt.binding_range(min=0, max=100, step=1, name='cutoff:')
-   selector = alt.selection_single(name="SelectorName", fields=['cutoff'],
-                                   bind=slider, init={'cutoff': 50})
+   selector = alt.selection_point(name="SelectorName", fields=['cutoff'],
+                                   bind=slider, value=[{'cutoff': 50}])
```

Comment:
I am not sure that we need this example anymore since it is easier to do this with parameter variables now (`selector = alt.parameter(value=50, bind=slider)`. But maybe it is good to show both next to each other, we could also move it higher up closer to / in the parametervariable section.

Reply:
Agreed - I don't think we would have added this example if variables had been available! Can you think of any reasons we might want to use this selection name lookup now that parameter variables are a thing? If not, I'd say we can remove this example.

Reply:
I remember I thought it was useful to learn that we could access the fields of parameters (as with `selector.cutoff` in that example), but I am not sure if that is useful. Maybe what I will do is to include it higher up as an example of some of the power introduced with variable in contrast to how we had to use selections before. I will ping when I have updated it.

Reply:
Done!